api/registrations/serializers.py

```diff
@@ -12,13 +12,12 @@ class RegistrationSerializer(NodeSerializer):
     retracted = ser.BooleanField(source='is_retracted', read_only=True,
                                  help_text='Whether this registration has been retracted.')
     date_registered = ser.DateTimeField(source='registered_date', read_only=True, help_text='Date time of registration.')
+    embargo_end_date = HideIfRetraction(ser.SerializerMethodField(help_text='When will embargo be lifted?'))
```

Comment:
Ah - I see.  `pending_registration_approval` is what I was calling the field in the serializer.  My source was wrong.  It was `sanction.pending_approval`, not `is_pending_registration`.  Apparently I don't understand sanctions.

If the source is corrected, then the values are different and everything works great. I will correct the source and restore the pending_embargo field.  Thanks @samchrisinger